ydb/library/yql/dq/actors/compute/dq_async_compute_actor.cpp

```diff
@@ -524,10 +523,6 @@ class TDqAsyncComputeActor : public TDqComputeActorBase<TDqAsyncComputeActor, TC
             }
         }
 
-        if (status != ERunStatus::Finished) {
```

Comment:
Is it true that this code is not necessary here?

Reply:
It used to call a method from TDqComputeActorBase, that does nothing